apps/beeai-cli/src/beeai_cli/commands/platform/configure_securityl.py

```diff
@@ -0,0 +1,440 @@
+                        }
+                    ],
+                },
+            }
+        ).encode("utf-8"),
+    )
+    await driver.run_in_vm(
+        [
+            "k3s",
+            "kubectl",
+            "--kubeconfig=/etc/rancher/k3s/k3s.yaml",
+            "apply",
+            "--server-side",
+            "-f",
+            "-",
+        ],
+        "Applying HTTPRoute CRD",
+        input=yaml.dump(
+            {
+                "apiVersion": "gateway.networking.k8s.io/v1",
+                "kind": "HTTPRoute",
+                "metadata": {"name": "beeai-platform-api"},
+                "spec": {
+                    "parentRefs": [{"name": "beeai-gateway", "namespace": "istio-system"}],
+                    "hostnames": ["beeai-platform.api.testing"],
+                    "rules": [
+                        {
+                            "matches": [{"path": {"type": "PathPrefix", "value": "/api/v1"}}],
+                            "backendRefs": [{"name": "beeai-platform-svc", "port": 8333}],
+                        }
+                    ],
+                },
+            }
+        ).encode("utf-8"),
+    )
+
+    await driver.run_in_vm(
+        [
+            "k3s",
+            "kubectl",
+            "--kubeconfig=/etc/rancher/k3s/k3s.yaml",
+            "apply",
+            "--server-side",
+            "-f",
+            "-",
+        ],
+        "Applying HTTPRoute CRD",
+        input=yaml.dump(
+            {
+                "apiVersion": "gateway.networking.k8s.io/v1",
+                "kind": "HTTPRoute",
+                "metadata": {"name": "beeai-platform-ui"},
+                "spec": {
+                    "parentRefs": [{"name": "beeai-gateway", "namespace": "istio-system"}],
+                    "hostnames": ["beeai-platform.testing", "beeai-platform.api.testing"],
+                    "rules": [
+                        {
+                            "matches": [{"path": {"type": "PathPrefix", "value": "/"}}],
+                            "backendRefs": [{"name": "beeai-platform-ui-svc", "port": 8334}],
+                        }
+                    ],
+                },
+            }
+        ).encode("utf-8"),
+    )
+
+    await driver.run_in_vm(
+        [
+            "/bin/sh",
+            "-c",
+            "curl -L https://istio.io/downloadIstio | sh -",
+        ],
+        "Installing istioctl...",
+    )
+
+    istio_dir = (
+        (
+            await driver.run_in_vm(
+                [
+                    "/bin/sh",
+                    "-c",
+                    "ls -lahF | grep istio | awk '{print $9}'",
+                ],
+                "Checking for istio install folder...",
+            )
+        )
+        .stdout.decode()
+        .strip()
+    )
+
+    await driver.run_in_vm(
+        [
+            "/bin/sh",
+            "-c",
+            f"export PATH=/{istio_dir}/bin:$PATH && k3s kubectl apply -f /{istio_dir}samples/addons/prometheus.yaml",
+        ],
```

Comment:
Why do you need `istioctl` in PATH for this? It seems that you only apply the YAML. You could skip downloading the full binary and just apply the YAMLs from GitHub.

Reply:
You can install from here, no need to install Istio for a few YAMLs https://raw.githubusercontent.com/istio/istio/refs/heads/master/samples/addons/prometheus.yaml